package.json

```diff
@@ -7,7 +7,9 @@
   "license": "MIT",
   "private": true,
   "workspaces": [
+    "packages/app",
     "packages/contracts",
+    "packages/landingpage",
```

Comment:
when I tried to push it under landing-page, it didn't show up on the pull request. I think it's because landing-page is a submodule? I've never worked with submodules so not sure the proper way to propose changes for them